### PR TITLE
Proposal: Consolidate ResultProvider resources

### PR DESCRIPTION
**Pull request by amcasey**
The language-specific resource files have only one entry each.  Just move
them into the shared resource file so that we will only need a single
satellite assembly.

NOTE: I will not be checking this in until after we RI because we will
likely need to update the RemoteTools setup authoring.

### CODE REVIEW

**Comment by amcasey** [comment]
FYI @tmat @pnelsonmsft @ManishJayaswal

**Comment by amcasey** [comment]
This will also save us (linked file) trouble when we split each ResultProvider dll into NetFX20 and Portable projects.

**Comment by KevinH-MS** [comment]
:+1:

File: src/ExpressionEvaluator/CSharp/Source/ResultProvider/CSharpResultProvider.csproj [hunk 1]
```diff
@@ -36,11 +36,6 @@
     <Compile Include="CSharpFormatter.Values.cs" />
```
Comment (VladimirReshetnikov) [inline src/ExpressionEvaluator/CSharp/Source/ResultProvider/CSharpResultProvider.csproj]: Consider adding `<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->` at the top of the file after the xml declaration.

File: src/ExpressionEvaluator/CSharp/Test/ResultProvider/CSharpResultProviderTest.csproj [hunk 2]
```diff
@@ -101,12 +101,6 @@
     <Compile Include="..\..\Source\ResultProvider\Helpers\TypeExtensions.cs">
```
Comment (VladimirReshetnikov) [inline src/ExpressionEvaluator/CSharp/Test/ResultProvider/CSharpResultProviderTest.csproj]: Consider adding `<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->` at the top of the file after the xml declaration.

File: src/ExpressionEvaluator/VisualBasic/Source/ResultProvider/BasicResultProvider.vbproj [hunk 3]
```diff
@@ -73,11 +73,6 @@
   <ItemGroup>
```
Comment (VladimirReshetnikov) [inline src/ExpressionEvaluator/VisualBasic/Source/ResultProvider/BasicResultProvider.vbproj]: Consider adding `<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->` at the top of the file after the xml declaration.

File: src/ExpressionEvaluator/VisualBasic/Test/ResultProvider/BasicResultProviderTest.vbproj [hunk 4]
```diff
@@ -86,12 +86,6 @@
     <Compile Include="..\..\Source\ResultProvider\Helpers\TypeExtensions.vb">
```
Comment (VladimirReshetnikov) [inline src/ExpressionEvaluator/VisualBasic/Test/ResultProvider/BasicResultProviderTest.vbproj]: Consider adding `<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->` at the top of the file after the xml declaration.

**Comment by amcasey** [comment]
Retest this please

**Pull request merged**